samples/open-telemetry/customizing/Core_8/CustomizingTelemetry/Sample/Behaviors/AddOrderIdToTrace.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NServiceBus.Pipeline;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+#region add-tags-from-behavior
+class AddOrderIdToTrace : Behavior<IIncomingLogicalMessageContext>
+{
+    public override Task Invoke(
+        IIncomingLogicalMessageContext context,
+        Func<Task> next
+    )
+    {
+        if(context.Message.Instance is ShipOrder shipOrder)
+        {
+            Activity.Current?.AddTag("sample.shipping.order_id", shipOrder.OrderId);
```

Comment:
setting message specific properties which is much easier later in the pipeline in this behavior feels weird. Could we add some more generic information, maybe the body size (more appropriate in the physical stage but that should be easy to change)?